moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h

```diff
@@ -62,8 +62,12 @@ class JogCalcs
 
   void stopMainLoop();
 
+  /** \brief Stop publishing jog commands while keeping the main loop spinning and filters up to date */
   void haltOutgoingJogCmds();
```

Comment:
To match with my comment below, a better name would be `pauseOutgoingJogCmds()`

Reply:
done